packages/devtools/src/composite.ts

```diff
@@ -550,16 +551,14 @@ export class Composite {
    * set as admin in the Ceramic node configuration must be attached to the Ceramic instance.
    */
   async startIndexingOn(ceramic: CeramicApi): Promise<void> {
-    const modelIndices: Map<string, Array<ModelIndex> | undefined> = new Map()
-    let definedIndices: Record<string, Array<ModelIndex>> = {}
-    if (this.#definition.indices && this.#definition.indices.models) {
-      definedIndices = this.#definition.indices.models
+    let definedIndices: Array<ModelFieldsIndex> = []
+    if (this.#definition.indices && this.#definition.indices.indices) {
+      definedIndices = this.#definition.indices.indices
     }
-    Object.keys(this.#definition.models).forEach((key) => {
-      const indices = definedIndices[key]
-      modelIndices.set(key.toString(), indices)
+    const modelIds = Array.from(Object.keys(this.#definition.models)).map((id) => {
+      return StreamID.fromString(id)
```

Comment:
Why are the model IDs parsed from string here please? They'll need to be converted to string in the API call.

Reply:
The http API deals in json, and thus uses strings. The API used by composedb is typescript, and uses StreamID. RemoteAdminAPI may use http underneath, but we shouldn't leak transport details to the user of the API.

Reply:
But that's an unnecessary change here. The client and APIs already work with strings, there no need to make such string (client) -> StreamID (here) -> string (transport) -> StreamID (server) conversions where we could just be string (client) -> string (transport - no change) -> StreamID (server). I expect the server also needs the models IDs as string anyway to interact with the DB, so really the conversion to StreamID doesn't bring anything to the flow.

Reply:
OK I guess we're doing these kinds of unnecessary conversions all over the place already so one more isn't going to hurt.